### ISSUE TITLE
Fails to recognize Yubikey NEO despite osx-path-ccid

### ISSUE THREAD

**Issue by pbronez** [Yubico/yubioath-desktop#31]
I installed Yubico Authenticator 2.1.0 via the [official OS X installer](https://developers.yubico.com/yubioath-desktop/Releases/) without incident. It launches properly, but doesn't recognize my Yubikey NEO. I hunted around, found the README, downloaded  `osx-patch-ccid`, ran the patch, and restarted. Still no luck.

When I run `osx-patch-ccid`, I get this output:

```
Patching file: /usr/libexec/SmartCardServices/drivers/ifd-ccid.bundle/Contents/Info.plist
Yubico Yubikey NEO OTP+CCID already present, skipping...
Yubico Yubikey NEO CCID already present, skipping...
Yubico Yubikey NEO U2F+CCID already present, skipping...
Yubico Yubikey NEO OTP+U2F+CCID already present, skipping...
Yubico Yubikey 4 CCID already present, skipping...
Yubico Yubikey 4 OTP+CCID already present, skipping...
Yubico Yubikey 4 U2F+CCID already present, skipping...
Yubico Yubikey 4 OTP+U2F+CCID already present, skipping...
Saving file: /usr/libexec/SmartCardServices/drivers/ifd-ccid.bundle/Contents/Info.plist
Patching complete!
```

This seems to suggest everything is ok, but the app still doesn't work. What am I missing here?

System:
OS X 10.10.3
Mid 2012 MacBook Pro
Python 2.7.9 :: Anaconda 2.2.0 (x86_64)

**Comment by pbronez** [comment]
By the way, the YubiKey Personalization Tool works just fine with the same key on the same system.

**Comment by dainnilsson** [comment]
What mode is the NEO in? It needs to have CCID enabled for Yubico Authenticator to be able to use it.
Can you try installing the YubiKey NEO Manager (https://developers.yubico.com/yubikey-neo-manager/Releases/) if it's not already installed, and make sure the key is detected there? That tool will also show you the mode information, as well as let you change the mode.

**Comment by pbronez** [comment]
Thanks for the pointer to the [YubiKey NEO Manager download page](https://developers.yubico.com/yubikey-neo-manager/Releases/). It took me a while to realize that it was different from the [Yubikey Personalization Tool](https://www.yubico.com/products/services-software/personalization-tools/use/). NEO Manager isn't linked on the main [Services & Software page](https://www.yubico.com/products/services-software/) and the [NEO product detail page](https://www.yubico.com/products/yubikey-hardware/yubikey-neo/) only links to the manual, not the download page. I don't think I would have found it without your link.

I was able to install YubiKey NEO Manager (v1.3.0). It recognized my YubiKey NEO right away and reported that the key was in `OTP + U2F` mode. I hit `change connection mode`, selected `CCID` in addition to `OTP` and `U2F`. I removed and re-inserted the key, and NEO Manager reported the key was now in `OTP + U2F + CCID` mode. I quit NEO Manager, removed my YubiKey, and launched Yubico Authenticator (v2.1.0). The application now worked properly.

I think it would be helpful for Authenticator to give a more informative error message in this situation. Something like "FYI YubiKey NEO won't work out of the box, you must download the NEO Manager and enable CCID mode"

Where does Authenticator store the seeds for the rotating codes? If it's local then the app should be able to see if anything has been saved, and if not display some basic configuration tips.
